src/docs/development/ui/layout/index.md

```diff
@@ -16,21 +16,22 @@ description: Learn how Flutter's layout mechanism works and how to build a layou
 {{site.alert.end}}
 
 This is a guide to building layouts in Flutter.
-You'll build the layout for the following screenshot:
+You'll build the layout for the following app:
 
-<img src="/docs/development/ui/layout/images/lakes.jpg" style="border:1px solid black" alt="finished lakes app that you'll build in 'Building a Layout'">
+{% include app-figure.md img-class="site-mobile-screenshot border"
+    image="ui/layout/lakes.jpg" caption="The finished app" %}
 
 This guide then takes a step back to explain Flutter's approach to layout,
 and shows how to place a single widget on the screen.
 After a discussion of how to lay widgets out horizontally and vertically,
 some of the most common layout widgets are covered.
 
-## Building a layout {#building}
+## Building a layout
```

Comment:
Did you check to make sure no one's depending on these anchors?

Reply:
Intrasite, yes. I removed these alternate heading anchors because they don't land the page in the proper place when navigated to. I'll was on the fence as to whether I should investigate further, but this seemed like the quickest fix. I'll open an issue.

Reply:
#1940